plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -3,6 +3,7 @@ package cisco_telemetry_mdt
 import (
 	"bytes"
 	"encoding/binary"
+        "encoding/json"
```

Comment:
Seems like formatting is mangled. Please use `make check` and `make fmt`. Also rebase your PR to current master to re-trigger the CI scripts.